hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/EncodingFacade.java

```diff
@@ -179,7 +198,9 @@ private FunctionResultBuilder functionResultBuilder() {
 		private String metadata;
 
 		private FunctionResultBuilder forFunction(final FunctionType functionType) {
-			if (functionType == FunctionType.MINT) {
+			if (functionType == FunctionType.CREATE) {
```

Comment:
Just a cosmetic detail, but is there a reason not to use `switch` given `FunctionType` is an `enum`?